core/data-test/src/main/kotlin/com/google/samples/apps/nowinandroid/core/data/test/networkmonitor/AlwaysOfflineNetworkMonitor.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples.apps.nowinandroid.core.data.test.networkmonitor
+
+import com.google.samples.apps.nowinandroid.core.data.util.NetworkMonitor
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flowOf
+
+class AlwaysOfflineNetworkMonitor : NetworkMonitor {
```

Comment:
`TestNetworkMonitor().apply { setConnected(false) }` is short enough, I don't think you need this.

Reply:
I didn't know ```TestNetworkMonitor``` exists. Thank you.